code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -56,12 +57,14 @@
 /obj/item/gun/energy/laser/carbine
 	name = "laser carbine"
 	desc = "A ruggedized laser carbine featuring much higher capacity and improved handling when compared to a normal laser gun."
```

Comment:
Improved handling?

Reply:
Not my description of the item but that's not a typo.